### PR TITLE
doc: add NOTICE and SECURITY files

### PR DESCRIPTION
**Pull request by ds-ext-akononenko**
Currently I still don't fully understand how to create DEPENDENCIES file properly, without removing dependencies. Still working on it.

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+The project maintains the following source code repositories
+in the GitHub organization https://github.com/catenax-ng:
+
+- https://github.com/catenax-ng/product-traceability-foss-frontend
+
+## Third-party Content
+
+This project leverages the following third party content.
+
+See yarn.lock file.
```
Comment (ds-ext-akononenko) [inline NOTICE.md:34]: in template it should mention DEPENDENCIES file, however as it's not here for now, I've mentioned yarn.lock as a source of truth for all third parties.

File: NOTICE.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Notices
```
Comment (evia-fbedarf) [inline NOTICE.md:1]: Where is the Apache-2.0 license file?

https://github.com/catenax-ng/foss-example/blob/main/general/LICENSE

File: NOTICE.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+This content is part of [CatenaX](https://catena-x.net).
+
+- Project home: https://github.com/catenax-ng
+
+See the AUTHORS file(s) distributed with this work for additional information regarding authorship.
```
Comment (evia-fbedarf) [inline NOTICE.md:7]: This line can be removed because we don't have an Authors file.

**Review by evia-fbedarf — CHANGES_REQUESTED** [review]

File: NOTICE.md (hunk 2 above)
Comment (ds-ext-akononenko) [reply NOTICE.md:1]: we already have it

File: NOTICE.md (hunk 3 above)
Comment (ds-ext-akononenko) [reply NOTICE.md]: changed

**Review by evia-fbedarf — APPROVED** [review]

**Pull request merged**